lib/bolt/config.rb

```diff
@@ -42,6 +42,12 @@ class Config
 
     PUPPETFILE_OPTIONS = %w[proxy forge].freeze
 
+    DEFAULT_CONFIG_PATHS = [
+      Bolt::Util.windows? ? 'C:\\Program Files\\Puppet Labs\\Bolt\\bolt.yaml' : '/etc/puppetlabs/bolt/bolt.yaml',
+      File.expand_path('.puppetlabs/etc/bolt/bolt.yaml', Bolt::Util.windows? ? ENV['USERPROFILE'] : '~'),
+      File.expand_path('.puppetlabs/bolt.yaml', Bolt::Util.windows? ? ENV['USERPROFILE'] : '~')
```

Comment:
This path was used by bolt in the 0.x series. I think its the correct choice for this but we should feature flag acutally merging in default config data based on whether the Boltdir or specified file has `future` set. We can then expose this as the only config behavior for Bolt 2.0.